packages/foreman/rubygem-rubyipmi/rubygem-rubyipmi.spec

```diff
@@ -75,6 +76,9 @@ cp -a .%{gem_dir}/* \
 %exclude %{gem_instdir}/Gemfile*
 
 %changelog
+* Fri Sep 06 2019 Eric D. Helms <ericdhelms@gmail.com> - 0.10.0-4
+- Updates to build for SCL
```

Comment:
I don't see any real updates in this file. You could clean the EL6 compatibility code if you want to make changes :)